test/test_unit_keystore_adapter.py

```diff
@@ -0,0 +1,43 @@
+# Builtin
+import unittest
+from unittest.mock import patch, call
+# Third-Party
+import keyring
+# Internal
+from conjur.wrapper import KeystoreAdapter
+
+CONJUR_URL = "https://conjur-hostname.com"
```

Comment:
their is already a constant in the constants.py that you can use called `TEST_HOSTNAME`

Reply:
Changed